app/src/main/kotlin/com/twain/interprep/presentation/ui/modules/dashboard/DashboardScreen.kt

```diff
@@ -1,48 +1,122 @@
+        floatingActionButton = {
+            FAB {
+                navController.navigate(AppScreens.AddInterview.route) {
+                    popUpTo(AppScreens.Dashboard.route)
+                }
+            }
+        },
+        content = { padding ->
+            LazyColumn(
+                modifier = Modifier.padding(padding),
+                verticalArrangement = Arrangement.spacedBy(16.dp)
+            ) {
+                item {
+                    Card(
+                        shape = Shapes.small,
+                        colors = CardDefaults.cardColors(containerColor = DashBoardBackgroundPurple)
+                    ) {
+                        Header(
+                            text = "Upcoming",
+                            textColor = DashBoardHeader,
+                            textStyle = MaterialTheme.typography.titleLarge,
+                            modifier = Modifier.padding(16.dp),
+                            fontWeight = FontWeight.Bold
+                        )
+                        LazyRow(
+                            modifier = Modifier.padding(start = 16.dp, bottom = 32.dp),
+                            horizontalArrangement = Arrangement.spacedBy(16.dp),
+                            verticalAlignment = Alignment.CenterVertically
+                        ) {
+                            items(5) { UpcomingInterviewCard() }
+                        }
                     }
                 }
-            },
-            content = { padding ->
-                Box(modifier = Modifier.padding(padding)) {}
+                item {
+                    Card(
+                        shape = Shapes.small,
+                        colors = CardDefaults.cardColors(containerColor = DashBoardBackgroundGreen)
+                    ) {
+                        Header(
+                            text = "Coming Next",
+                            textColor = DashBoardHeader,
+                            textStyle = MaterialTheme.typography.titleLarge,
+                            modifier = Modifier.padding(16.dp),
+                            fontWeight = FontWeight.Bold
+                        )
+                        LazyRow(
+                            modifier = Modifier.padding(start = 16.dp, bottom = 32.dp),
+                            horizontalArrangement = Arrangement.spacedBy(16.dp)
+                        ) {
+                            items(5) { ComingNextInterviewCard() }
+                        }
+                    }
+                }
+                item {
+                    Header(
+                        text = "Past",
+                        textColor = DashBoardHeader,
+                        textStyle = MaterialTheme.typography.titleLarge,
+                        modifier = Modifier.padding(start = 16.dp),
+                        fontWeight = FontWeight.Bold
+                    )
+                }
+                items(25) { PastInterviewCard() }
```

Comment:
To fix the scrolling, I wrap each horizontal scrollable list inside a item(){} of a `LazyColumn` composable and list the vertical scrollable `PastInterviewCard` directly inside the `LazyColumn` instead of wrap them in a card, but one problem of this is that I can't specify the background color of the the list of pastInterviewCards.

Reply:
That's okay. We will revisit this implementation in future. We are not going to put any separate color for each 3 layout. I have updated the figma file. But card elevation issue is fixed and check my updated implementation.